docs/commands/bundle.md

```diff
@@ -169,15 +168,15 @@ All other information is omitted.
 You may want to skip specific preprocessors, rules, or decorators upon running the command.
 
 ```bash Skip preprocessors
-redocly bundle --skip-preprocessor=discriminator-mapping-to-one-of,another-example
+redocly bundle --skip-preprocessor=discriminator-mapping-to-one-of --skip-preprocessor=another-example
 ```
 
 ```bash Skip rules
-redocly bundle --skip-rule=no-sibling-refs,no-parent-tags
```

Comment:
I wander if we can support backward-compatibility here? What do we receive as a value for `skip-rule` arg from the parser? `no-sibling-refs,no-parent-tags`? Maybe we can split it manually?

Reply:
It never worked this way :(